lighthouse-core/gather/gatherers/forms.js

```diff
@@ -0,0 +1,185 @@
+  const labelsArray = [];
+
+  const childrenArray = Array.prototype.slice.call(formElement.childNodes);
+  for (const element of childrenArray) {
+    if (element.nodeName === 'LABEL') {
+      /** @type { {id: any; nodeName: any; name: any; for: any;} } */
+      const labelAttributes = {
+        id: element.id,
+        nodeName: element.nodeName,
+        name: element.name,
+        for: element.htmlFor,
+      };
+      labelsArray.push(labelAttributes);
+    }
+  }
+
+  return labelsArray;
+}
+
+/**
+ * @param {HTMLElement[]} formChildren
+ * @return { { inputs: { id: any; nodeName: any; name: any; placeholder: any; autocomplete: any; }[], labels: { id: any; nodeName: any; name: any; for: any; }[] } }
+ */
+/* istanbul ignore next */
+function getFormlessElements(formChildren) {
+  /** @type { { inputs: { id: any; nodeName: any; name: any; placeholder: any; autocomplete: any; }[], labels: { id: any; nodeName: any; name: any; for: any; }[] } } */
+  const formless = {
+    inputs: [],
+    labels: [],
+  };
+
+  for (const childElement of formChildren) {
+    if (childElement instanceof HTMLInputElement || childElement instanceof HTMLTextAreaElement
+      || childElement instanceof HTMLLabelElement || childElement instanceof HTMLSelectElement) {
+      if (childElement.form ) continue;
+    }
+
+    if (childElement instanceof HTMLInputElement || childElement instanceof HTMLTextAreaElement ) {
+      const inputAttributes = {
+        id: childElement.id,
+        nodeName: childElement.nodeName,
+        name: childElement.name,
+        placeholder: childElement.placeholder,
+        autocomplete: childElement.autocomplete,
+      };
+      formless.inputs.push(inputAttributes);
+    } else if (childElement instanceof HTMLSelectElement ) {
+      const selectAttributes = {
+        id: childElement.id,
+        nodeName: childElement.nodeName,
+        name: childElement.name,
+        placeholder: childElement.getAttribute('placeholder'),
+        autocomplete: childElement.autocomplete,
+      };
+      formless.inputs.push(selectAttributes);
+    } else if (childElement instanceof HTMLLabelElement) {
+      const labelAttributes = {
+        id: childElement.id,
+        nodeName: childElement.nodeName,
+        name: childElement.getAttribute('name'),
+        for: childElement.htmlFor,
+      };
+      formless.labels.push(labelAttributes);
+    }
+  }
+
+  return formless;
+}
+
+
+/**
+ * @return {LH.Artifacts['Forms']}
+ */
+/* istanbul ignore next */
+function collectFormElements() {
+  // @ts-ignore - put into scope via stringification
+  const formElements = getElementsInDocument('form'); // eslint-disable-line no-undef
+  // @ts-ignore - put into scope via stringification
+  const inputs = getElementsInDocument('input'); // eslint-disable-line no-undef
+  // @ts-ignore - put into scope via stringification
+  const labels = getElementsInDocument('labels'); // eslint-disable-line no-undef
```

Comment:
```suggestion
  const labels = getElementsInDocument('label'); // eslint-disable-line no-undef
```

Reply:
note that there are no `LABEL` in the test expectations b/c of this typo

Reply:
+1 some label smoke tests both wrapping and not wrapping inputs would be awesome 👍